ui/widgets/nuxeo-date.js

```diff
@@ -70,6 +72,10 @@ import './nuxeo-tooltip.js';
         tooltipFormat: String,
       };
     }
+
+    _sameFormat(datetime, format, tooltipFormat) {
```

Comment:
Since it is a boolean method, I would suggest `_hasSameFormat` for being aligned with boolean method names best practices 🙂

Reply:
nit: You're actually comparing the result so the function name currently does not describe the behavior

Reply:
Indeed!

Reply:
hasSameFormatResult?

Reply:
renamed to _producesSameDateFormat